android/src/main/java/org/rncloudfs/RNCloudFsModule.java

```diff
@@ -401,8 +401,10 @@ public void requestSignIn() {
     }
 
     @ReactMethod
-    public void getCurrentlySignedInUserData(Promise promise) {
-        Log.d(TAG, "Logging out");
+    public void getCurrentlySignedInUserData(Promise promise, ReadableMap options) {
```

Comment:
```suggestion
    public void getCurrentlySignedInUserData(ReadableMap options, Promise promise) {
```
Promise always needs to be the last param, otherwise the app crashes on runtime like this:

FATAL EXCEPTION: mqt_native_modules
Process: me.rainbow, PID: 12611
java.lang.AssertionError: Promise must be used as last parameter only